File: app/src/main/java/com/alvinhkh/buseta/kmb/model/KmbRouteBound.java
package com.alvinhkh.buseta.kmb.model;

import com.google.gson.annotations.SerializedName;

public class KmbRouteBound {

  @SerializedName("SERVICE_TYPE")
  public Integer serviceType;

  @SerializedName("BOUND")
  public Integer bound;

  @SerializedName("ROUTE")
  public String route;
}

File: app/src/main/java/com/alvinhkh/buseta/kmb/model/KmbRouteBound.kt
package com.alvinhkh.buseta.kmb.model

import com.google.gson.annotations.SerializedName

data class KmbRouteBound(
        @SerializedName("SERVICE_TYPE")
        var serviceType: Int = 0,

        @SerializedName("BOUND")
        var bound: Int = 0,

        @SerializedName("ROUTE")
        var route: String = ""
)